### PR TITLE
language in portuguese

### PR DESCRIPTION
**Pull request by onix-comunicacoes**

### CODE REVIEW

**Comment by onix-comunicacoes** [comment]
language in portuguese

File: apps/condo/lang/pt.json [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+  "ticket.status.DECLINED.name": "Recusado",
+  "ticket.status.COMPLETED.name": "Concluído",
+  "ticket.status.DEFERRED.name": "Deferido",
+  "ticket.status.CLOSED.name": "Fechado",
+  "TicketAssignment": "Atribuição de aplicativo",
+  "ticketsTable.Number": "№",
+  "ticketsTable.ResidentName": "Nome do morador",
+  "ticket.shareTitle": "Ticket №{número} datado de {data}",
+  "ticket.shareDetails": "O texto do ticket: \"{details}\"",
+  "ticket.shareWarningEmail": "Alguns funcionários não receberão 0 ticket porque não temos seus e-mail. Você pode adicionar e-mails na seção \"{link}\".\nNenhum e-mail para:\n{employees}\ n",
```
Comment (AleX83Xpert) [inline apps/condo/lang/pt.json:750]:
```suggestion
  "ticket.shareWarningEmail": "Alguns funcionários não receberão 0 ticket porque não temos seus e-mail. Você pode adicionar e-mails na seção \"{link}\".\nNenhum e-mail para:\n{employees}\n",
```

File: apps/condo/lang/pt.json [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+  "filters.NewTemplateLabel": "Nome do modelo (Por favor, preencha-o apenas se você deseja salvar o modelo. Para aplicar um filtro, deixe este campo em branco.)",
+  "filters.NewTemplatePlaceholder": "ou exemplo, \"Ingressos pagos no Moscow Residential Com plex\"",
+  "filters.DeleteTitle": "Excluir modelo de filtro?",
+  "filters.DeleteMessage": "Seria impossível restaurar o conjunto de configurações de filtros. Em vez disso, você pode criar um novo e salvá-lo.",
+  "filters.SaveTemplate": "Salvar modelo e aplicar",
+  "filters.FindWord": "Encontrar palavra",
+  "filters.FullSearch": "Pesquisar por qualquer campo do aplicativo",
+  "filters.Reset": "Redefinir Filtros",
+  "filters.UserName": "Nome Completo",
+  "FooterText": "Tem alguma dúvida? Envie um e-mail para {email}\ ou ligue para {phone}",
```
Comment (AleX83Xpert) [inline apps/condo/lang/pt.json:117]:
```suggestion
  "FooterText": "Tem alguma dúvida? Envie um e-mail para {email}\nou ligue para {phone}",
```

**Review by toplenboren — APPROVED** [review]

**Comment by onix-comunicacoes** [comment]
approved

**Comment by AleX83Xpert** [comment]
Hello @onix-comunicacoes!
Thanks for you taking a part in our project.
We did not forget about your pull request. I should notice that these two files do not contain all translations. We also have a number of templates (email, SMS, push) that are hardcoded for now.
My current task is a deep refactoring of our internationalization system. The result will be all data need for translation collected in a single folder.
After refactoring will finished, you may continue your great contribution.

**Comment by AleX83Xpert** [comment]
Hi @onix-comunicacoes!
We've finished refactoring of translation subsystem. Now you can continue to translate.
Also, we create small [documentation about translations](https://github.com/open-condo-software/condo/blob/master/docs/localization.md).

**Pull request closed without merging**